tests/unit/core/agents/agent_components/test_agent_config.py

```
@@ -0,0 +1,223 @@
+    _coerce_global_request_timeout,
+    _coerce_optional_str,
+    _coerce_request_delay,
+    _compute_agent_version,
+    _get_provider_config_from_registry,
+    _ProviderConfig,
+    load_system_prompt,
+)
+
+
+@pytest.fixture(autouse=True)
+def _clear_caches():
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_clear_caches`

(ANN202)